core/src/utilities/helpers/generic-helpers.js

```diff
@@ -316,6 +313,44 @@ class GenericHelpersClass {
     }
     return val;
   }
+
+  createRemotePromise() {
+    let res, rej;
+    const prom = new Promise((resolve, reject) => {
+      res = () => {
+        resolve();
+      };
+      rej = () => {
+        reject(new Error());
```

Comment:
When I try to open a route twice I'm getting an error in the console
![Bildschirmfoto 2022-02-10 um 08 57 49](https://user-images.githubusercontent.com/14272184/153368594-7797ff88-1d8a-4883-b570-c59d29bccbdc.png)

Reply:
If unsaved changes warning modal is displayed and dismissed I'll get the error too.
You can reproduce it when you go to _overview_ in our e2e test app and turn on the toggle _There are unsaved changes_. Try to navigate somewhere else like in the attached screenshot and click on _somewhere else_ and dismiss the warnings modal.

![Bildschirmfoto 2022-02-14 um 16 21 26](https://user-images.githubusercontent.com/14272184/153892613-e2808116-981e-42aa-8522-c99e5db5a52a.png)

Reply:
Unfortunately I can reproduce it. I merged and resolved all conflict between this branch and `master`  so maybe now there is't this error.